sys-fs/mergerfs/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>ahippo@yandex.ru</email>
+	</maintainer>
```

Comment:
Who's gonna commit for you?

Reply:
I'll add proxy-maint as with the other PR.
Somehow missed this in the PR/Proxy_Maint instructions.

Reply:
(becoming a Gentoo developer would be cool too, but I hope the PR will get merged sooner than that)